src/main/java/frc4388/robot/subsystems/Apriltags.java

```diff
@@ -0,0 +1,49 @@
+package frc4388.robot.subsystems;
+
+//import edu.wpi.first.apriltag.AprilTag;
+//import edu.wpi.first.math.geometry.Pose3d;
+//import edu.wpi.first.math.geometry.Rotation3d;
+//import edu.wpi.first.networktables.NetworkTable;
+//import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.networktables.NetworkTableInstance;
+
+public class Apriltags {
+    public static class Tag {
+        boolean visible = true;
+        double x, y, z = 0;
+    }
+    
+    public Object[] getApriltagPosition() {
```

Comment:
This Object[] stuff isn't the worst but it does lose a lot of what makes a static typing system so valuable. I added a tag class above that we could use.